modules/andes-core/broker/src/main/java/org/wso2/andes/amqp/QpidAndesBridge.java

```diff
@@ -168,46 +168,14 @@ private QpidAndesBridge() {
      * This should happen after all content chunks are received
      *
      * @param incomingMessage  message coming in
-     * @param channelID        id of the channel message came in
      * @param andesChannel     AndesChannel
      * @param transactionEvent not null if this is a message in a transaction, null otherwise
-     * @throws AMQException
+     * @throws AMQException when routing key is null
      */
-    public static void messageReceived(IncomingMessage incomingMessage, UUID channelID,
-                           AndesChannel andesChannel, InboundTransactionEvent transactionEvent) throws AMQException {
```

Comment:
AndesChannel is removed from the signature of the method. Does this mean distributed transaction impl does  not have flow controlling?